charts/nfdiv-frontend/values.preview.template.yaml

```diff
@@ -8,6 +8,11 @@ nodejs:
     primaryIngressHost: ${SERVICE_FQDN}
   autoscaling:
     enabled: false
+  keyVaults:
+    nfdiv:
+      resourceGroup: nfdiv
+      secrets:
+        - AppInsightsInstrumentationKey-Preview
```

Comment:
do you not require other secrets here ? https://github.com/hmcts/nfdiv-frontend/blob/7467555d537b05d5809f3bc98b6833b2dcaa6ed7/charts/nfdiv-frontend/values.yaml#L27-L31